packages/nvidia_nat_a365/tests/tooling/test_function_group.py

```
@@ -0,0 +1,155 @@
+from nat.builder.function import Function
+from nat.builder.function import FunctionGroup
+from nat.plugins.a365.tooling import A365MCPToolingConfig
+from nat.plugins.a365.tooling.register import A365MCPToolingFunctionGroup
+
+
+@pytest.fixture(name="base_config")
+def base_config_fixture():
+    """Base config for tests."""
+    return A365MCPToolingConfig(
+        agentic_app_id="test-agent",
+        auth_token="test-token",
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to argument: "auth_token"

(S106)